docs/tutorials/blinky/pinetime.rst

```diff
@@ -0,0 +1,187 @@
+
+    $ newt target show
+    targets/blinky-pinetime
+        app=apps/blinky
+        bsp=@apache-mynewt-core/hw/bsp/pinetime
+        build_profile=debug
+    targets/boot-pinetime
+        app=@mcuboot/boot/mynewt
+        bsp=@apache-mynewt-core/hw/bsp/pinetime
+        build_profile=optimized
+
+Build the Target Executables
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Run the ``newt build boot-pinetime`` command to build the bootloader:
+
+.. code-block:: console
+
+    $ newt build boot-pinetime
+    Building target targets/boot-pinetime
+    Compiling bin/targets/boot-pinetime/generated/src/boot-pinetime-sysflash.c
+    Compiling bin/targets/boot-pinetime/generated/src/boot-pinetime-sysinit-app.c
+    Compiling bin/targets/boot-pinetime/generated/src/boot-pinetime-sysdown-app.c
+    Compiling repos/mcuboot/boot/bootutil/src/boot_record.c
+    Compiling repos/mcuboot/boot/bootutil/src/caps.c
+
+        ...
+
+    Archiving sys_sysinit.a
+    Archiving util_mem.a
+    Archiving util_rwlock.a
+    Linking /tmp/blinky-pinetime/bin/targets/boot-pinetime/app/boot/mynewt/mynewt.elf
+    Target successfully built: targets/boot-pinetime
+
+
+Run the ``newt build blinky-pinetime`` command to build the Blinky
+application:
+
+.. code-block:: console
+
+    $ newt build blinky-pinetime
+    Building target targets/blinky-pinetime
+    Compiling bin/targets/blinky-pinetime/generated/src/blinky-pinetime-sysinit-app.c
+    Compiling bin/targets/blinky-pinetime/generated/src/blinky-pinetime-sysdown-app.c
+    Compiling bin/targets/blinky-pinetime/generated/src/blinky-pinetime-sysflash.c
+    Compiling repos/apache-mynewt-core/hw/bsp/pinetime/src/sbrk.c
+    Compiling apps/blinky/src/main.c
+    Assembling repos/apache-mynewt-core/hw/bsp/pinetime/src/arch/cortex_m4/gcc_startup_nrf52.s
+    Compiling repos/apache-mynewt-core/hw/bsp/pinetime/src/hal_bsp.c
+
+        ...
+
+    Archiving sys_sysinit.a
+    Archiving util_mem.a
+    Archiving util_rwlock.a
+    Linking /tmp/blinky-pinetime/bin/targets/blinky-pinetime/app/apps/blinky/blinky.elf
+    Target successfully built: targets/blinky-pinetime
+
+Sign and Create the Blinky Application Image
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Run the ``newt create-image blinky-pinetime 1.0.0`` command to create and
+sign the application image. You may assign an arbitrary version (e.g.
+1.0.0) to the image.
+
+.. code-block:: console
+
+    $ newt create-image blinky-pinetime 1.0.0
+    App image succesfully generated: ~/dev/blinky-pinetime/bin/targets/blinky-pinetime/app/apps/blinky/blinky.img
+
+Connect to the Board
+~~~~~~~~~~~~~~~~~~~~
+
+-  Connect the ST-LINK programmer to the SWD connectors of the PineTime. The 
+   easiest way it to mount the PineTime on a cradle and let the programmer cable 
+   rest on the contact pads. See `PineTime wiki <https://wiki.pine64.org/index.php/PineTime#SWD_Pinout>`__ 
+   and `this article <https://github.com/lupyuen/visual-embedded-rust/blob/master/README.md>`__ 
```

Comment:
Can you remove the link for that README and add a schematic? It doesn't make much sense to point to that article if the only information we need is how to connect the debugger.

Reply:
I tried to add a ASCII pinout table, that didn't give a good result. So I removed the README article and left the Wiki link for the pinout. Hope that is fine with you, else I will remove the links all together.